components/tidb_query_expr/src/impl_compare_in.rs

```diff
@@ -196,6 +198,58 @@ where
     }
 }
 
+#[rpn_fn(nullable, varg, capture = [metadata], min_args = 1, metadata_mapper = init_compare_in_data::<T>)]
+#[inline]
```

Comment:
It seems that `compare_in_int_type_by_hash_fn_meta` is only used for `Int`. Can we remove generics in this function?

Reply:
use Int instread of T as generic type